release-please-config.json

```diff
@@ -0,0 +1,19 @@
+{
+    "packages": {
+      ".": {
+        "package-name": "science-portal",
+        "changelog-path": "CHANGELOG.md",
+        "release-type": "simple",
+        "bump-minor-pre-major": false,
+        "bump-patch-for-minor-pre-major": false,
+        "draft": false,
+        "prerelease": false,
+        "include-component-in-tag": false,
+        "include-v-in-tag": true,
```

Comment:
This will push the tag as `v0.4.0`. Then in the release-build actions, you also need to add the following step to create a sanitized tag, if you want to release it as `0.4.0`

```yaml
        name: Sanitize Release Tag
        id: sanitize-tag
        run: |
          echo "Sanitizing Tag: ${{ env.TAG_RELEASE }}"
          export SANITIZED_TAG_RELEASE=$(echo ${{ env.TAG_RELEASE }} | sed 's/^v//'
          echo "Sanitized Tag: $SANITIZED_TAG_RELEASE"
          echo SANITIZED_TAG_RELEASE=$SANITIZED_TAG_RELEASE >> $GITHUB_ENV
```         
and then add it to tags field in the docker/buildx action.

Reply:
Done, thanks!